### PR TITLE
Configure ocis collaboration service in docker compose

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
## Description
Set up collaboration service (wopi server provided by oCIS) in docker compose.

## Related Issue
- Part of https://github.com/owncloud/web/issues/10960

## How Has This Been Tested?
- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

### CODE REVIEW

**Review by dschmidt — COMMENTED** [review]
If I understood Michael correctly this should replace the old wopiserver, correct? Then that should be removed.
Also we have Collabora and OnlyOffice in the docker compose setup a collaboration service should be used for each probably?

File: docker-compose.yml [hunk 1]
```diff
@@ -224,6 +224,37 @@ services:
       - host.docker.internal:${DOCKER_HOST:-host-gateway}
     restart: always
 
+  collaboration:
```
Comment (saw-jan) [inline docker-compose.yml:227]: we also need separate service configs for onlyoffice and collabora. And we have to think how to do it. via `.env` or `profiles`

**Comment by saw-jan** [comment]
> If I understood Michael correctly this should replace the old wopiserver, correct? Then that should be removed. Also we have Collabora and OnlyOffice in the docker compose setup a collaboration service should be used for each probably?

yeah, that is true.

**Comment by saw-jan** [comment]
>Also we have Collabora and OnlyOffice in the docker compose setup a collaboration service should be used for each probably?

seems like we can use both office suites at the same time by starting two different collaboration services on different ports (`COLLABORATION_GRPC_ADDR` and `COLLABORATION_HTTP_ADDR`)

**Review by saw-jan — APPROVED** [review]
LGTM

**Review by ScharfViktor — APPROVED** [review]
awesome. Thank you 👍

**Pull request merged**